<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-AP49OGSV" ignored="0" name="RetryMulitple" tests="6" failures="1" timestamp="27 Sep 2018 11:36:04 GMT" time="124.065" errors="0">
  <testcase name="extentFail" time="0.018" classname="reTryMechanism.LoginPage">
    <skipped/>
  </testcase> <!-- extentFail -->
  <testcase name="extentFail" time="0.001" classname="reTryMechanism.LoginPage">
    <skipped/>
  </testcase> <!-- extentFail -->
  <testcase name="extentFail" time="0.002" classname="reTryMechanism.LoginPage">
    <failure type="java.lang.AssertionError" message="Failed Test expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed Test expected [true] but found [false]
at reTryMechanism.LoginPage.extentFail(LoginPage.java:90)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- extentFail -->
  <testcase name="verifyLogin" time="8.533" classname="reTryMechanism.LoginPage">
    <skipped/>
  </testcase> <!-- verifyLogin -->
  <testcase name="verifyLogin" time="65.392" classname="reTryMechanism.LoginPage"/>
  <testcase name="verifyLoginPage" time="0.636" classname="reTryMechanism.LoginPage"/>
</testsuite> <!-- RetryMulitple -->
